java/arcs/core/data/expression/Stringifier.kt

```diff
@@ -12,74 +12,81 @@ package arcs.core.data.expression
 
 /** Traverses a tree of [Expression] objects, converting each node to manifest format (String). */
 class ExpressionStringifier(val parameterScope: Expression.Scope = ParameterScope()) :
-    Expression.Visitor<String> {
+    Expression.Visitor<String, Unit> {
```

Comment:
Should we provide a super class `ContextFreeVisitor<T> : Visitor<T, Unit>` or similar? It would probably simplify a lot of visitors we have that don't need the context.

Reply:
Let me do in a follow on cleanup if possible, touching the visitor APIs sadly creates a lot of diff churn.